CMakeLists.txt

```diff
@@ -363,6 +460,7 @@ hardware/PhilipsHue/PhilipsHueSensors.cpp
 hardware/PiFace.cpp
 hardware/Pinger.cpp
 hardware/PVOutput_Input.cpp
+hardware/RAVEn.cpp
```

Comment:
I dont think this should be in the pull request, as also the below (xiamigateway)
Maybe you have to bring your forked repository a little up to date ?